source/common/chromium_url/url_canon.h

```diff
@@ -28,7 +28,7 @@ namespace chromium_url {
 // manage.
 template <typename T> class CanonOutputT {
 public:
-  CanonOutputT() : buffer_(NULL), buffer_len_(0), cur_len_(0) {}
+  CanonOutputT() : buffer_(NULL) {}
```

Comment:
s/NULL/nullptr/ and also make this an initializer like `*_len_` -- but only if we are going to continue to apply clang-tidy to this Chromium code.